src/core/event/index.js

```diff
@@ -18,14 +18,20 @@ export function Events(Base) {
 
       // If 'history', rely on the browser's scroll auto-restoration when going back or forward
       if (source !== 'history') {
+        let focusEl;
+
         // Scroll to ID if specified
         if (this.route.query.id) {
-          this.#scrollIntoView(this.route.path, this.route.query.id);
+          focusEl = dom.find(`#${this.route.query.id}`);
+          this.#scrollIntoView(this.route.path, this.route.query.id, true);
         }
         // Scroll to top if a link was clicked and auto2top is enabled
-        if (source === 'navigate') {
+        else if (source === 'navigate') {
+          focusEl = dom.find('.content');
           auto2top && this.#scroll2Top(auto2top);
         }
+
+        focusEl && focusEl.focus();
```

Comment:
This make every page navigation focus on content. Is it standard for `shift+tab` to be the way to navigate to navigation (sidebar)? What are some real-world examples?

Reply:
> This makes every page navigation focus on content after the sidebar.

Having the focus move to the `<body>` element on each new page load would more closely mimic the behavior of a "standard" where clicking a link reload the entire page. This will allow users to quickly access the content area using the "Skip to main content" link or continue on to the navigation after each page load.

I will make this change.

> Is it standard for shift+tab to be the way to navigate to navigation (i.e. move backwards through the sidebar)?

Not to my knowledge. Regardless, the change described above will remove this concern.